project/Http4sPlugin.scala

```diff
@@ -185,9 +183,8 @@ object Http4sPlugin extends AutoPlugin {
         releaseStepCommand("test:scalafmt::test").when(primary),
         releaseStepCommand("docs/makeSite").when(primary),
         releaseStepCommand("website/makeSite").when(primary),
-        openSonatypeRepo.when(publishable && release),
-        publishToSonatypeWithoutInstrumentation.when(publishable),
-        releaseAndClose.when(publishable && release),
+        releaseStepCommandAndRemaining("+publishSigned").when(publishable),
+        releaseStepCommand("sonatypeReleaseAll").when(publishable && release),
```

Comment:
Introduces slight race conditions across repositories. I've never run into it in practice though.